benchmarks/torchbench/benchfile.py

```diff
@@ -54,19 +54,20 @@ def install(self):
                 "ind = indices.detach().cpu().numpy()"
             )
 
-        self.pip_install("-r", code / "requirements-bench.txt")
         model_name = self.config["model"]
 
+        reqs = None
         if headless and (code / f"requirements-{model_name}-headless.txt").exists():
-            self.pip_install("-r", code / f"requirements-{model_name}-headless.txt")
+            reqs = f"requirements-{model_name}-headless"
         elif (code / f"requirements-{model_name}.txt").exists():
-            self.pip_install("-r", code / f"requirements-{model_name}.txt")
-        if (code / f"noinstallpy-{model_name}.txt").exists():
-            # We don't want to use the model's install.py, but we still want to
-            # run the global install.py, so we pass resnet18 which has an empty
-            # install.py.
-            self.python("install.py", "--models", "resnet18")
-        else:
+            reqs = f"requirements-{model_name}"
+
+        if reqs:
+            self.pip_install("-r", code / f"{reqs}.txt")
+            if (code / f"{reqs}-extra.txt").exists():
+                self.pip_install("-r", code / f"{reqs}-extra.txt")
```

Comment:
What are these?

Reply:
Name is probably quite bad but this is specifically for `requirements-detectron2_maskrcnn-extra.txt` which contains `detectron2` for which the `setup.py` requires `torch` to be installed : https://github.com/facebookresearch/detectron2/blob/c470ca31d2b54d038b9e78f48d1d8e1a8debcc68/setup.py